frontend/src/pages/answer/Answer.jsx

```diff
@@ -192,13 +216,98 @@ export default function Answer() {
     event.target.value = '';
   }
 
+  /**
+   * Save an uploaded answer to Robokache
+   */
+  async function saveAnswer() {
+    let accessToken;
+    if (isAuthenticated) {
+      try {
+        accessToken = await getAccessTokenSilently();
+      } catch (err) {
+        displayAlert('error', `Failed to validate user. Error: ${err}`);
```

Comment:
```suggestion
        displayAlert('error', `Failed to authenticate user. Error: ${err}`);
```

Reply:
Updated in 06dfb063189bcbd4f9ce23d3ace08fba1c9301f6